src/Checkbox/types.ts

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+export interface CheckboxProps extends React.HTMLAttributes<HTMLInputElement> {
+  checked?: boolean;
```

Comment:
Checked is optional, because checkbox can be indeterminate and when checkbox is indeterminate it should not be checked.

Reply:
But when indeterminate is true, checked can be false. And still, have a value no? I am still not sure how indeterminate works, I read that need to be done from JS because can not be done in HTML.

Reply:
A checkbox can have three possible states: checked, unchecked or indeterminate. I think someone can define a checkbox with only indeterminate prop, without checked prop.